src/pudl/transform/ferc1.py

```diff
@@ -99,21 +100,25 @@ class Ferc1RenameColumns(TransformParams):
     xbrl: RenameColumns = {}
 
 
-class WideToTidyXbrl(TransformParams):
-    """Parameters for converting a wide XBRL table to a tidy table with value types."""
+class WideToTidyDataSource(TransformParams):
```

Comment:
Are we using this `DataSource` suffix in other transformers that can work on either DBF or XBRL inputs? If not should we be? Or should we not be doing it here?

Reply:
the other param that is similar to this is `Ferc1RenameColumns`... which I think goes against our existing convention of putting FERC1 at the end.

I used `DataSource` because it was close to `source_ferc1: Ferc1Source`. perhaps I should: 
- rename this param to be `WideToTidyFerc1Source`
- rename `Ferc1Source` to `SourceFerc1`

Reply:
there are a fair amount of things named `Ferc1xxxxx`.... i'm a little tempted to rename them all but i'm not sure what your original intention was around that.